testsuite/features/secondary/allcli_software_channels_dependencies.feature

```diff
@@ -23,11 +23,11 @@ Feature: Channel subscription with recommended or required dependencies
     Then I should see the child channel "SLE-Product-SLES15-SP4-Updates for x86_64" "selected" and "disabled"
     When I exclude the recommended child channels
     Then I should see the toggler "disabled"
-    And I should see a "SLE-Module-Containers15-SP4-Pool for x86_64" text
-    And I should see the child channel "SLE-Module-Containers15-SP4-Pool for x86_64" "unselected"
+    And I should see a "Desktop Applications Module 15 SP4 x86_64" text
+    And I should see the child channel "Desktop Applications Module 15 SP4 x86_64" "unselected"
     # check the a child channel selection that requires some channel trigger the selection of it
-    When I select the child channel "SLE-Module-Containers15-SP4-Pool for x86_64"
-    Then I should see the child channel "SLE-Module-Containers15-SP4-Pool for x86_64" "selected"
+    When I select the child channel "Desktop Applications Module 15 SP4 x86_64"
+    Then I should see the child channel "Desktop Applications Module 15 SP4 x86_64" "selected"
```

Comment:
Why do we sync the desktop applications instead?

Reply:
This test only use a channel to play with, this scenario is not about synchronizing anything.
Any previously selected channel would work as example.